File: code_blocks/integrations/attribution/branch_1.dart
import 'package:flutter_branch_sdk/flutter_branch_sdk.dart';
import 'package:purchases_flutter/purchases_flutter.dart';

// login
FlutterBranchSdk.setIdentity('my_app_user_id');

// Optional: Use a different App User ID between Branch and RevenueCat
Purchases.setAttributes({'\$branchId': '<custom_branch_user_id>'});

// logout
FlutterBranchSdk.logout();

File: code_blocks/integrations/attribution/branch_1.kt
import io.branch.referral.Branch
import com.revenuecat.purchases.Purchases

// login
Branch.getInstance().setIdentity("my_app_user_id")

// Optional: Use a different App User ID between Branch and RevenueCat
Purchases.sharedInstance.setAttributes(mapOf("\$branchId" to "<custom_branch_user_id>"))

// logout
Branch.getInstance().logout()
